proxy/proxy.go

```diff
@@ -1449,7 +1449,6 @@ func (p *Proxy) Close() error {
 func (p *Proxy) setCommonSpanInfo(u *url.URL, r *http.Request, s ot.Span) {
 	p.tracing.
 		setTag(s, ComponentTag, "skipper").
-		setTag(s, HTTPUrlTag, u.String()).
```

Comment:
you can replace with log instead

Reply:
Replacing the tag with a log is not going to help reduce the size of the gRPC message